actionview/test/template/form_helper_test.rb

```diff
@@ -3256,6 +3290,12 @@ def test_fields_for_with_index
     assert_dom_equal expected, output_buffer
   end
 
+  def test_fields_for_field_id_with_index_option
+    fields_for(:post, @post, index: 5) do |fields|
```

Comment:
```suggestion
    fields_for(:post, @post) do |fields|
```

Otherwise if I undo the index-related implementation changes, the tests still pass.

Reply:
Oh! Thank you for catching that. I've pushed up your suggested change and squashed the commits.